src/scripts.js

```diff
@@ -54,25 +54,27 @@ const addIndividualUserInfo = () => {
 
 const submitNewTripRequest = (event) => {
   event.preventDefault()
-  // if (domUpdates.resolveTripRequest()) {
-    const tripRequest = {
-      id: Number(trips.data.length + 1),
-      userID: Number(user.id),
-      destinationID: Number(destinations.retrieveDestinationID(destinationInput.value)),
-      travelers: Number(numberOfTravelersInput.value),
-      date: dayjs(dateInput.value).format('YYYY/MM/DD'),
-      duration: Number(durationInput.value),
-      status: 'pending',
-      suggestedActivities: [],
-    }
-    domUpdates.resolveTripRequest(trips.retrieveTripCost(destinations.data, tripRequest).toFixed(2));
-    // domUpdates.displayPendingTrips(user, destinations.data);
-    addData(tripRequest, 'trips')
-      .then(data => updatePendingTrips(data), 'data')
-      .catch(err => console.log(err, "error"))
-  // }
+  if (dateInput.value && durationInput.value && numberOfTravelersInput.value && destinationInput.value) {
+  const tripRequest = {
+    id: Number(trips.data.length + 1),
```

Comment:
Love this to make the trip id instead of doing date.now or some other randomizer type thing.